### ISSUE TITLE
Different Pre-trained Models

### ISSUE THREAD

**Issue by Yeez-lee** [zyxxmu/lottery-jackpots#2]
Dear authors,

Thanks for sharing this great work!
I want to try different training strategies to train my own ResNet32 or VGG16 but there are some bugs because of your modified conv layers by mask. Could you share the codes about how you train these networks to get the dense pre-trained checkpoints?

Thanks!

**Comment by zyxxmu** [comment]
Please kindly refer to https://github.com/kuangliu/pytorch-cifar
